subsys/net/lib/openthread/platform/uart.c

```diff
@@ -208,7 +210,14 @@ otError otPlatUartSend(const u8_t *aBuf, u16_t aBufLength)
 	size_t cnt = ring_buf_put(ot_uart.tx_ringbuf, aBuf, aBufLength);
 
 	if (atomic_set(&(ot_uart.tx_busy), 1) == 0) {
-		uart_irq_tx_enable(ot_uart.dev);
+		if(is_panic_mode){
```

Comment:
fix spaces in this block - space after if, before bracket and after 'else'

Reply:
Done